tests/test_e2e_21_flow_manager.py

```diff
@@ -138,3 +138,64 @@ def test_031_on_switch_restart_kytos_should_recreate_flows(self):
         flows_s1 = s1.dpctl('dump-flows')
         assert len(flows_s1.split('\r\n ')) == BASIC_FLOWS + 1, flows_s1
         assert 'dl_vlan=999' in flows_s1
+
+    def test_032_deserialization_by_restarting_kytos(self):
```

Comment:
This test case is really great. Can we give it a slightly more meaningful name of what is testing though? Maybe`test_032_on_switch_reconnection_should_recreate_untagged_any_flows`?

Reply:
Changed.